#download reference genome, make sure star, bedtools is installed (e.g., conda install -c bioconda star, conda install -c bioconda bedops)
cd GRCh38
wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_43/GRCh38.primary_assembly.genome.fa.gz
wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_43/gencode.v43.annotation.gtf.gz
gunzip *.gz
STAR --runThreadN 70 \
     --runMode genomeGenerate \
     --genomeDir GRCh38_STAR_index \
     --genomeFastaFiles GRCh38.primary_assembly.genome.fa \
     --sjdbGTFfile gencode.v43.annotation.gtf
     
#STAR only gives chromosome location: we can annotate those location with bedtools (but it needs a bed file and not gtf)
#therefore 1.) transform gtf into bed
# name the feature here, we filter only columns with this faeture!!
gtf2bed --attribute-key=gene_name < data/GRCh38/gencode.v43.annotation.gtf > data/GRCh38/genes.bed
#filter out chromosome region plus the feature
cut -f1-4 data/GRCh38/genes.bed | sort -u > data/GRCh38/genesFiltered.bed

#download HELA data of SIGNAL-seq (multimodal data for RNA&protein generated by combinatorial indexing)
cd ../SIGNALseq
wget "https://sra-pub-run-odp.s3.amazonaws.com/sra/SRR28056729/SRR28056729" -O SIGNALseq/SRR28056729.sra
wget "https://sra-pub-run-odp.s3.amazonaws.com/sra/SRR28056728/SRR28056728" -O SIGNALseq/SRR28056728.sra
fastq-dump --split-files --gzip SIGNALseq/SRR28056729.sra
fastq-dump --split-files --gzip SIGNALseq/SRR28056728.sra
# processed single-cell RNA/ protein data for the hela dataset can be downloaded here: https://zenodo.org/records/10676523


# download xDBiT (spatial single-cell data with x&y coordinate barcodes)
